airflow/providers/papermill/operators/papermill.py

```diff
@@ -17,17 +17,102 @@
 # under the License.
 from __future__ import annotations
 
+from functools import cached_property
 from typing import TYPE_CHECKING, ClassVar, Collection, Sequence
 
 import attr
+from papermill.utils import remove_args, merge_kwargs
+from pydantic import typing
+
 import papermill as pm
+from papermill.engines import NBClientEngine
+from papermill.clientwrap import PapermillNotebookClient
+from jupyter_client.manager import AsyncKernelManager
+from jupyter_client.client import KernelClient
+from traitlets import Unicode
 
 from airflow.lineage.entities import File
 from airflow.models import BaseOperator
+from airflow.providers.papermill.hooks.kernel import KernelHook
 
 if TYPE_CHECKING:
     from airflow.utils.context import Context
 
+REMOTE_KERNEL_ENGINE = "remote_kernel_engine"
```

Comment:
Could you explain why it should be constant value?

Reply:
We are registering a remote kernel engine(custom papermill engine) so the operator can work with remote kernels.